src/Assets/TestProjects/TestAppEchoDotnetRoot/Program.cs

```diff
@@ -2,15 +2,18 @@
 // Licensed under the MIT license. See LICENSE file in the project root for full license information.
 
 using System;
+using System.Runtime.InteropServices;
 
 namespace ConsoleApplication
 {
     public class Program
     {
         public static void Main(string[] args)
         {
+            var processArchitecture = $"DOTNET_ROOT_{RuntimeInformation.ProcessArchitecture.ToString().ToUpper()}";
```

Comment:
Does this work in globalization invariant mode?

Reply:
I guess this is just a test so it probably doesn't matter.

Reply:
Note that this is also done in RunCommand. I changed to `Enum.GetName` in both places.

Reply:
This will change the casing though - will the host work correctly with that?
I think on Windows environment variables are case insensitive, while on Linux they are case sensitive. The host asks for upper case - always. So I think this will effectively break on mac, right?

Would it be a problem to use ToUpperInvariant() instead?